src/extensions/extra/MDNDocs/README.md

```diff
@@ -0,0 +1,7 @@
+# Updating the Docs
+Use the Node script scrape-mdn in `/tools` or clone repo from [scrape-mdn](http://github.com/jordantheriault/scrape-mdn) to update the `css.json` and 'html.json' contents:
```

Comment:
remove `clone repo from [scrape-mdn](http://github.com/jordantheriault/scrape-mdn) to update the `css.json` and 'html.json' contents`